e2e/rbd_helper.go

```diff
@@ -167,9 +168,25 @@ func createRBDStorageClass(
 		sc.MountOptions = append(sc.MountOptions, mOpt...)
 	}
 	sc.ReclaimPolicy = &policy
-	_, err = c.StorageV1().StorageClasses().Create(context.TODO(), &sc, metav1.CreateOptions{})
 
-	return err
+	timeout := time.Duration(deployTimeout) * time.Minute
+
+	return wait.PollImmediate(poll, timeout, func() (bool, error) {
+		_, err = c.StorageV1().StorageClasses().Create(context.TODO(), &sc, metav1.CreateOptions{})
+		if err != nil {
+			e2elog.Logf("error creating StorageClass %q in namespace %q: %v", sc.Name, sc.Namespace, err)
```

Comment:
storageclass is not a namespaced object, remove the namespace from it?

Reply:
yeah, but it is part of the CephFS function too, this is just a copy/paste

Reply:
Maybe we can correct it both places in the same PR or in a different PR?

Reply:
this was done, btw